src/pages/ThorChainLP/utils.ts

```diff
@@ -1,3 +1,4 @@
+import * as Sentry from '@sentry/react'
```

Comment:
Likewise on this boi, named imports pleez

Reply:
e6f70eb16bc3414ecc3c18df860b030e9d11e5ee

Reply:
And obviously tested at runtime and confirmed Sentry still happy